core/src/Platforms/iOS/WebviewBase.cs

```diff
@@ -42,6 +42,10 @@ internal abstract class WebviewBase : NSObject, IWebUI, ISFSafariViewControllerD
         protected SFAuthenticationSession sfAuthenticationSession;
         protected ASWebAuthenticationSession asWebAuthenticationSession;
 
+        protected nint taskId = UIApplication.BackgroundTaskInvalid;
+        protected NSObject didEnterBackgroundNotification;
+        protected NSObject willEnterForegroundNotification;
```

Comment:
Seems like you are no longer disposing these guys. It should probably be ensured that happens.